doc/SensorFusion/SRS/srs_eng.md

```diff
@@ -0,0 +1,372 @@
+# Software Requirements Specification (SRS)
+## Project Title: Sensorfusion of Localization data for Pololu Zumo32U4
+
+Version 0.2
+Author: Tobias Haeckel
+Bachelor Thesis
+
+## Version History
+| Name | Date  | Reason for Change  | Version |
+|------|--------|---------------------|---------|
+|Tobias Haeckel | 2025-10-01 |Init|0.1|
+
+## Table of Contents
+- [Software Requirements Specification (SRS)](#software-requirements-specification-srs)
+  - [Project Title: Sensorfusion of Localization data for Pololu Zumo32U4](#project-title-sensorfusion-of-localization-data-for-pololu-zumo32u4)
+  - [Version History](#version-history)
+  - [Table of Contents](#table-of-contents)
+- [1. Introduction](#1-introduction)
+  - [1.1 Purpose](#11-purpose)
+  - [1.2 Product Scope](#12-product-scope)
+  - [1.3 Definitions, Acronyms, and Abbreviations](#13-definitions-acronyms-and-abbreviations)
+  - [1.4 References](#14-references)
+  - [1.5 Overview](#15-overview)
+- [2. Overall Description](#2-overall-description)
+  - [2.1 Product Perspective](#21-product-perspective)
+  - [2.2 Product Features](#22-product-features)
+  - [2.3 Assumptions and Dependencies](#23-assumptions-and-dependencies)
+    - [2.3.1 Assumptions](#231-assumptions)
+    - [2.3.2 Dependencies](#232-dependencies)
+- [3. Requirements](#3-requirements)
+  - [3.1 External Interface Requirements](#31-external-interface-requirements)
+    - [3.1.1 User Interfaces](#311-user-interfaces)
```

Comment:
Maybe going 3 levels deep is too much for the table of contents. Maybe even 2 levels deep is already too much information

Reply:
Maybe the auto-generation can be tailored, or left completely out?